ftw/simplelayout/opengraph/og_site_root.py

```diff
@@ -42,9 +42,24 @@ def _settings(self):
         return registry.forInterface(
             ISimplelayoutDefaultSettings, check=False)
 
+    def _escape(self, text):
+        """More or less taken from https://wiki.python.org/moin/EscapingHtml
+        """
+        html_escape_table = OrderedDict([
```

Comment:
If we already do it manually we could make it stronger by using pythons inbuilt modules:
```python
>>> import cgi
>>> data = u'<this is="ä">test'
>>> cgi.escape(data, quote=True).encode('ascii', 'xmlcharrefreplace')
'&lt;this is=&quot;&#228;&quot;&gt;test'
```

`cgi.escape` does the same as you are doing here and the `.encode(..)` additionally also makes sure that umlauts are escaped.

Reply:
@Nachtalb it does not handle `&`. and this is one of the official proposed solution from the https://wiki.python.org/moin/EscapingHtml 

I considered using cgi modul to but then I still had to do the & manually. IMHO this one is the most explicit solution. 

Umlauts do not need to be encoded.